fuel-p2p/Cargo.toml

```diff
@@ -13,14 +13,18 @@ description = "Fuel client networking"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-trait = "0.1.52"
+bincode = "2.0.0-rc.1"
```

Comment:
Should we be using an rc for this dep?

Reply:
We already use bincode in other parts of fuel-core so we should use the same version here for compiling efficiency as well.